plugin/trino-base-jdbc/src/test/java/io/trino/plugin/jdbc/BaseJdbcConnectorTest.java

```diff
@@ -2017,6 +2020,19 @@ private void assertNoDynamicFiltering(@Language("SQL") String sql)
     }
 
     private void assertDynamicFiltering(@Language("SQL") String sql, JoinDistributionType joinDistributionType, boolean expectDynamicFiltering)
+    {
+        assertEventually(
+                new Duration(10, MINUTES),
```

Comment:
10 minutes is a lot

Reply:
I remember we had 
```
Method io.trino.plugin.jdbc.BaseJdbcConnectorTest.testDynamicFiltering() didn't finish within the time-out 60000
```
that's why I've decided 10 minutes.